lib/bpk-component-text-input/src/BpkTextInput.js

```diff
@@ -124,11 +125,11 @@ type State = {
 
 type EnhancedProps = Props & WithBpkAppearanceInjectedProps;
 
+const createTinyMask = memoize(mask => new TinyMask(mask || ''));
```

Comment:
I believe it might be good to avoid having the same memoize cache for all the instances of BpkTextInput, as cache will be constantly overwritten by different instances of BpkTextInput.

Perhaps createTinyMask should be declared as a BpkTextInput method, this will ensure that separate memoize cache is created per component instance.

Reply:
The cache is per "mask" string, so there is no issue in using the same, it's actually beneficial to do so as all mask strings will share the same instance of `TinyMask`.